paddle/fluid/framework/ir/mkldnn/elt_act_mkldnn_fuse_pass.cc

```diff
@@ -17,44 +17,25 @@
 #include "paddle/fluid/framework/ir/graph_pattern_detector.h"
 #include "paddle/fluid/framework/op_version_registry.h"
 #include "paddle/fluid/platform/enforce.h"
+#include "paddle/fluid/platform/mkldnn_reuse.h"
 #include "paddle/fluid/string/pretty_log.h"
 
 namespace paddle {
 namespace framework {
 namespace ir {
 
+using paddle::platform::GetAttributeMap;
+using paddle::platform::GetSupportedActivations;
 using string::PrettyLogDetail;
 
 void ElementwiseActivationOneDNNPass::ApplyImpl(Graph *graph) const {
-  std::vector<std::string> act_types = {"relu",
-                                        "tanh",
-                                        "leaky_relu",
-                                        "swish",
-                                        "hard_swish",
-                                        "sqrt",
-                                        "abs",
-                                        "clip",
-                                        "gelu",
-                                        "relu6",
-                                        "sigmoid"};
+  auto act_types = GetSupportedActivations();
   std::vector<std::string> elt_types = {
       "elementwise_add", "elementwise_sub", "elementwise_mul"};
 
   for (const auto &elt_type : elt_types)
     for (const auto &act_type : act_types) {
-      std::unordered_map<std::string, std::string> attr_map;
-
-      if (act_type == "swish")
-        attr_map.emplace("beta", "fuse_alpha");
-      else if (act_type == "relu6")
-        attr_map.emplace("threshold", "fuse_alpha");
-      else if (act_type == "clip") {
-        attr_map.emplace("min", "fuse_alpha");
-        attr_map.emplace("max", "fuse_beta");
-      } else {
-        attr_map.emplace("alpha", "fuse_alpha");
-        attr_map.emplace("beta", "fuse_beta");
-      }
+      auto attr_map = GetAttributeMap(act_type);
       FuseElementwiseAct(graph, elt_type, act_type, attr_map);
```

Comment:
You can reduce number of parameters by moving attr_map extraction inside FuseElementwiseAct function.
Let's do this here and in other passes.
```suggestion
      FuseElementwiseAct(graph, elt_type, act_type);
```